README.md

```diff
@@ -164,23 +83,23 @@ d3.json(url).then(data => {
 });
 ```
 
-Another example can be found [here](./application/static/js/buildDataTable.js).
+- You can build on our work by creating your own visualizations using the API. Examples for how to integrate the data retrieved from the API into visualizations including charts and maps can be found in [`static/js`](./static/js).
 
-## Rendering the bushfire map locally
+#### Rendering the bushfire map 
 
-To view the live bushfire map on your own machine, you'll need to create and save a `config.py` file in the root directory within the **application** folder of this repository with a Mapbox API key included in the following format:
+To view the live bushfire map on your own machine, you'll need to create and save a `config.py` file in the root directory within the **application** folder of this repository that includes a Mapbox API key in the following format:
 
 ```py
 API_KEY = "your key here"
 ```
 
 More information on Mapbox API keys can be found on [their website](https://docs.mapbox.com/help/how-mapbox-works/access-tokens/).
 
-## Deployment
+### Step 5. Deploy the site to Heroku
```

Comment:
Jumps from step 3 to step 5. Maybe just have deployment as a separate section since it is not part of installation.